lib/directory_diff/transform.rb

```diff
@@ -40,7 +40,10 @@ def process_employee(email, assistant_owner)
 
       if new_employee.nil?
         add_transform(:delete, old_employee)
-        assistant_owner[3] = nil
+        assistants_string = assistant_owner[3].to_s.split(",").reject do |assistant|
+                              assistant == email
+                            end.join(",")
+        assistant_owner[3] = assistants_string == "" ? nil : assistants_string
```

Comment:
`presence` is a rails thing, but yeah that's exactly what i was trying to simulate